lexicon/client.py

```diff
@@ -10,6 +11,9 @@
     legacy_config_resolver, non_interactive_config_resolver,
 )
 
+TLDEXTRACT_CACHE_FILE_DEFAULT = os.path.join(os.path.dirname(__file__), '.tld_set')
+TLDEXTRACT_CACHE_FILE = os.path.expanduser(os.environ.get("LEXICON_TLDEXTRACT_CACHE",
```

Comment:
Good catch to invoke `os.path.expanduser` here!

Reply:
That one was taken directly from tldextract 😉